data/simulations/FAH_SARS-CoV-2_spike.yml

```diff
@@ -0,0 +1,40 @@
+type: md
+title: Trajectories of the SARS-CoV-2 spike protein from Folding@Home
+description: >
+  All-atom MD simulations of the SARS-CoV-2 spike protein, simulated using Folding@Home. 
```

Comment:
Is there a preprint that goes with this? If so, would be good to cite it!

Reply:
I have the preprint cited with the `preprint` tag. Should I put it in the description too?